init.js

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env node
+'use strict';
+const execa = require('execa');
+const npmRunPath = require('npm-run-path');
+
+const env = npmRunPath.env({cwd: __dirname});
+
+execa('alfred-link', {env}).catch(err => {
```

Comment:
Nah, I can't come up with any real use-case for that, except for ours, which is solved better this way.